static/js/composebox_typeahead.js

```diff
@@ -733,6 +739,14 @@ exports.get_candidates = function (query) {
             }
         }
     }
+    if (this.options.completions.timestamp) {
+        const time_jump_regex = /!time(\(([^\)]*?))?$/;
```

Comment:
Why is this named `time_jump` -- is this in analogy to the `topic_jump` feature?

Reply:
Yes, that was the idea: we're jumping the logic to some other typeahead/autocomplete instead of the current typeahead being the final piece.